sphinx_doc/build_instructions/build.rst

```diff
@@ -0,0 +1,235 @@
+
+  source ~/nav2_depend_ws/install/setup.bash
+  mkdir -p ~/navigation2_ws/src
+  cd ~/navigation2_ws/src
+  git clone https://github.com/ros-planning/navigation2.git --branch master
+  cd ~/navigation2_ws
+  rosdep install -y -r -q --from-paths src --ignore-src --rosdistro eloquent
+  colcon build --symlink-install
+
+Step 4- Turtlebot3 (Optional)
+=============================
+
+Turtlebot3 is one of the target robot platforms to run Navigation2. You can skip these build instructions for Turtlebot3 packages if Turtlebot3 is not your target platform.
+
+- Install Turtlebot 3
+
+.. code:: bash
+
+  sudo apt install ros-<ros2-distro>-turtlebot3-*
+
+**Recommended:** To get the lastest Turtlebot3 packages and build them from source, follow the instructions in this `E-Manual <http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2_setup>`_
+
+* Make sure you select the correct ROS 2 distribution when following the build steps.
+
+  ``ex: sudo apt install ros-<ros2-distro>-<package-name>``
+
+**Tutorials**
+
+After successfully building Navigation2 from source, you can test it by following `these tutorials <https://github.com/ros-planning/navigation2/tree/master/doc/tutorials>`_
+
+Quickstart using initial setup script
+*************************************
+
+Steps
+=====
+
+  - Install all ROS 2 dependencies from the `ROS2 Installation page <https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Development-Setup>`_
+
+  - Ensure there are no ROS environment variables set in your terminal or `.bashrc` file before taking the steps below.*
+
+
+.. code:: bash
+
+  mkdir <directory_for_workspaces>
+  cd <directory_for_workspaces>
+  wget https://raw.githubusercontent.com/ros-planning/navigation2/master/tools/initial_ros_setup.sh
+  chmod a+x initial_ros_setup.sh
+  ./initial_ros_setup.sh
+
+
+**Summary of what's being done**
+
+The ``initial_ros_setup.sh`` script downloads four ROS workspaces and then builds them in the correct order. The four workspaces are:
+
+- **ROS 2 release**: This is the latest ROS 2 release as defined by the repos file found `here <https://github.com/ros2/ros2>`_
+- **ROS 2 dependencies**: This is a set of ROS 2 packages that aren't included in the ROS 2 release yet. However, you need them to be able to build Navigation2. This also includes packages that are part of the ROS 2 release where Navigation2 uses a different version.
+- **Navigation2**: This repository.
+
+After all the workspaces are downloaded, run the `navigation2/tools/build_all.sh` script. `build_all.sh` builds each repo in the order listed above using the `colcon build --symlink-install` command.
+
+Options
+=======
+
+The `initial_ros_setup.sh` accepts the following options:
+
+- `--no-ros2` This skips downloading and building the ROS 2 release. Instead it uses the binary packages and ``setup.sh`` installed in ``/opt/ros/<ros2-distro>``
+- ``--download-only`` This skips the build steps
+
+Use Case Recommendations
+------------------------
```

Comment:
```suggestion
************************
```

This looks like a subsection under the quickstart instructions. This should be at the top level

Reply:
I used "=" instead to move the title one level up. The navigation menu looks like this now.

![build_ins](https://user-images.githubusercontent.com/12270197/70347697-f7438a80-1815-11ea-9767-3092c20cf9a2.png)